hc/accounts/tests/test_profile.py

```diff
@@ -29,6 +29,15 @@ def test_it_sends_report(self):
 
```

Comment:
Also, where are the tests that test whether a user can configure daily, weekly or monthly reports?
@pndemo